### PR TITLE
3 rewrite login desktop

### LINKED ISSUE #3: Re-write: Login page
Remove the absolute attribute in CSS 
Have read throw and cleaned up the old code
Adapt the design to the later version. 
make it responsive

### PR DESCRIPTION
**Pull request by ESPR07**
Rewritten the login page to closely match the figma design, It's now made with responsiveness in mind, but Mobile version is not implemented yet.

Closes #3

### CODE REVIEW

File: src/components/Login.tsx [hunk 1]
```diff
@@ -33,15 +33,23 @@ function LoginComponent() {
         <CssBaseline />
         <Box
           sx={{
-            marginTop: 8,
             display: "flex",
             flexDirection: "column",
             alignItems: "center",
           }}
         >
           <Typography component="h1" variant="h5">
```
Comment (AKissMail) [inline src/components/Login.tsx:41]: You could fix the display issue with 
```html
<h1 className="signupH1">Log in to Projects Without Borders</h1>

```

File: src/styles/login.css [hunk 2]
```diff
@@ -1,55 +1,72 @@
...
-.imgLogo{
-position: relative;
-bottom: 10%;
-margin: auto;
-width: 50%;
+.loginText,
+.signupText {
+  color: var(--mainGreen);
+  font-size: min(3vw, 1.7rem);
+  font-weight: bolder;
```
Comment (AKissMail) [inline src/styles/login.css:32]: What about 800 for the font weight and we have a global setting for the font family in main.css 
```css
.loginText,
.signupText {
  color: var(--mainGreen);
  font-size: min(3vw, 1.7rem);
  font-weight: 800;
  align-items: center;
  margin-inline: 30px;
  font-family: var(--mainFontFamily);
}

```

File: src/styles/login.css [hunk 3]
```diff
@@ -1,55 +1,72 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
+  flex-basis: 60%;
 }
-.loginText>p, .signupText>p{
-    margin: 0;
 
+.signupRedirect {
```
Comment (AKissMail) [inline src/styles/login.css:56]: Here I also suggest to uses the global property a bit more font weight
```css
.signupRedirect {
  width: 90%;
  text-align: end;
  font-family: var(--mainFontFamily);
  font-weight: bolder;
}
```

File: src/styles/login.css [hunk 4]
```diff
@@ -1,55 +1,72 @@
...
 
-.loginPart, .signupPart{
-    margin-left: 40%;
+.signupLink:hover {
+  text-decoration: underline;
+}
+
+main {
+  margin: auto;
 }
```
Comment (AKissMail) [inline src/styles/login.css:72]: New class needed if you accept my suggestion for the heading: 
```css
.signupH1 {
  font-family: var(--mainFontFamily);
  font-size: 25pt;
}
```

**Review by AKissMail — CHANGES_REQUESTED** [review]
Hi, I located some cosmetic issues. I left you some comments.

File: src/components/Login.tsx (hunk 1 above)
Comment (ESPR07) [reply src/components/Login.tsx]: This has been added and fixed :D

**Review by AKissMail — COMMENTED** [review]
Nice!

**Review by AKissMail — APPROVED** [review]

**Pull request merged**